sdkproject/Assets/Mapbox/Unity/Map/AbstractMap.cs

```diff
@@ -661,6 +662,173 @@ public void SetLoadingTexture(Texture2D loadingTexture)
 			Options.loadingTexture = loadingTexture;
 			_mapVisualizer.SetLoadingTexture(loadingTexture);
 		}
+
+		#region Location Prefabs Methods
+
+		/// <summary>
+		/// Places a game object prefabs at the specified LatLon on the Map.
+		/// </summary>
+		/// <param name="prefab"> A Game Object Prefab.</param>
+		/// <param name="LatLon">A Vector2d(Latitude Longitude) object</param>
+		public void SpawnPrefabAtGeoLocation(GameObject prefab, Vector2d LatLon, Action<List<GameObject>> callback = null, bool scaleDownWithWorld = true, string locationItemName = "New Location")
+		{
+			var coordinateArray = new string[] { LatLon.x + ", " + LatLon.y };
```

Comment:
This method body can be simplified to `SpawnPrefabAtGeoLocation(prefab, coordinateArray,....)`

Reply:
I think it's better to take a Vector2d in the method parameters and handle the string conversion. 

We're limited to strings in the UI and sublayer properties to keep things compatible with the `GeoCodeAttributeSearchWindow` for now, but in script it's easier to work with lat/lon as a vector2d than a comma separated string.